accumulate/accumulate-test.rkt

```diff
@@ -0,0 +1,20 @@
+#lang racket
+
+(require "accumulate.rkt")
+
+(module+ test
+  (require rackunit rackunit/text-ui)
+  
+  (define suite
+    (test-suite
+     "accumulate tests"
+     
+     (test-equal? "empty list"
+                  (accumulate empty (lambda (arg) (arg)))
```

Comment:
The `(arg)` in the body of the `lambda` here, attempts to call it as a function with no arguments, which surely isn't your intention.